docs/_sidebar.md

```diff
@@ -52,13 +52,12 @@
   - [Scaling](/wiki/research/scaling/scaling.md)
     - [Core Changes](/wiki/research/scaling/core-changes/core-changes.md)
     - [EIP-4844](/wiki/research/scaling/core-changes/eip-4844.md)
-    - Blobs
     - Sharding
     - Statelessness
     - Purge
-  - MEV
-    - PBS
-    - Censorship
+  - [MEV](/wiki/research/PBS/mev.md)
```

Comment:
May be it is good idea to add a Glossary at the root level of the wiki and add a one sentence summary and cross reference the wiki pages.

Reply:
What do you suggest..?

Reply:
We have to add a `Glossary.md` in the root directory and add a one line explanations the abbreviations and other important terms and concepts. This is for the whole wiki not only PBS.

May be we can do that once we merge this PR to main branch.